src/main/java/ca/intelliware/ihtsdo/mlds/web/rest/CommercialUsageResource.java

```diff
@@ -460,8 +459,8 @@ private void synchronize() {
     @RolesAllowed({AuthoritiesConstants.STAFF,AuthoritiesConstants.ADMIN })
     @Timed
     @Transactional
-    public @ResponseBody Collection<Collections> reviewUsageReportCsv(){
-        Collection collections=commercialUsageRepository.findUsageReport();
+    public @ResponseBody Collection<Object[]> reviewUsageReportCsv(){
+        Collection<Object[]> collections=commercialUsageRepository.findUsageReport();
```

Comment:
@dmcgihtsdo We have changed the response type this method because it was not compatible with latest SB3. We have also tested and confirmed that there is no issue on the UI Side.